client/src/components/Notices/LegendSymbols.js

```diff
@@ -0,0 +1,31 @@
+import QuestionCircle from 'react-icons/lib/fa/question-circle'
+import ExclamationTriangle from 'react-icons/lib/fa/exclamation-triangle'
+import classnames from 'classnames'
+
+import React from 'react'
+
+export function getWarningSymbol(level) {
+  if (level < 0) {
+    return <QuestionCircle className={classnames('warning', `warning${level}`)} />
+  } else if (level > 0) {
+    return <ExclamationTriangle className={classnames('warning', `warning-${level}`)} />
+  } else {
+    return ''
+  }
+}
+
+export function formatSimilarPercent(value) {
+  let style = ''
+  if (value > 75) {
```

Comment:
const styles = [{style: 'similarity-high', num: 75}, {style: 'similarity-medium', num: 50}, {style: 'similarity-low', num: 25}, {style: '', num: -1}}]
return styles.map((style) => value > style.num).style

You can probably brainstorm better names though..